routers/intake.py

```
@@ -58,6 +59,42 @@ async def intake_message(
 
     parsed = parse_user_input(raw_text, message.entities)
     status_message = await message.reply(text.PROCESSING)
+        # === TERABOX EXPERIMENTAL BYPASS API ===
+    if any(domain in parsed.source_url for domain in ["terabox", "1024tera", "teraboxapp"]):
+        logger.info(f"TeraBox link detected! Intercepting pipeline for bypass...")
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)